gen/testdata/thrift/constants.thrift

```diff
@@ -0,0 +1,86 @@
+include "./other_constants.thrift"
+include "./containers.thrift"
+include "./enums.thrift"
+include "./exceptions.thrift"
+include "./structs.thrift"
+include "./unions.thrift"
+
+const containers.PrimitiveContainers primitiveContainers = {
+    "listOfInts": other_constants.listOfInts, // imported constant
```

Comment:
This is a valid comment as per the [Thrift scanner](https://github.com/apache/thrift/blob/775671aea41ea55427dd78d7ce68e282cc9b8487/compiler/cpp/src/thriftl.ll#L116). Not sure how to make GH happy.